velox/docs/functions/presto/binary.rst

```diff
@@ -101,3 +101,19 @@ Binary Functions
 .. function:: xxhash64(binary) -> varbinary
 
     Computes the xxhash64 hash of ``binary``.
+    
+.. function:: lpad(binary, size, padbinary) -> varbinary
```

Comment:
These descriptions are in alphabetical order. Please move these to the right position in that list.